ipi/engine/outputs.py

```diff
@@ -290,7 +289,7 @@ def __init__(
            cell_units: A string specifying the units that the cell parameters are
               given in.
            ibead: If positive, prints out only the selected bead. If negative, prints out one file per bead.
-           xtratype: Specifies the type of extras string that is printed in the file
+           extra_type: Specifies the type of extras string that is printed in the file
```

Comment:
Should we add that type is used-defined?

Reply:
why not. and that it defaults to printing the full "raw" string